Tests/SwiftStdlibTests/KeyedDecodingContainerTests.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  KeyedDecodingContainerTests.swift
+//  SwifterSwift
+//
+//  Created by Francesco Deliro on 23/10/2019.
+//  Copyright © 2019 SwifterSwift
+//
+
+import XCTest
+
+@testable import SwifterSwift
+
+private struct Video: Decodable {
+    var isPlaying: Bool
+    var isFullScreen: Bool?
+
+    enum CodingKeys: String, CodingKey {
+        case isPlaying
+        case isFullScreen
+    }
+
+    init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        isPlaying = try container.decodeBoolAsIntOrString(forKey: .isPlaying)
+        isFullScreen = try container.decodeBoolAsIntOrStringIfPresent(forKey: .isFullScreen)
+    }
+}
+
+private enum ElementToTest: Decodable {
```

Comment:
You can remove this and simplify all the test strings you have by getting rid of the `data` root object, instead just use the internal value.